google/auth/compute_engine/__init__.py

```diff
@@ -11,3 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+"""Google Compute Engine authentication."""
+
+from google.auth.compute_engine.credentials import Credentials
+
+
+__all__ = [
+    'Credentials'
+]
```

Comment:
> Are you suggesting to just move credentials into here?

No. I'm suggesting `_all__` be removed since `Credentials` is already the only thing in the namespace.